### ISSUE TITLE
X25519Kyber768Draft00 named group not being picked up by boringssl static

### ISSUE THREAD

**Issue by jmcrawford45** [netty/netty-tcnative#863]
I am setting 
`System.setProperty("jdk.tls.namedGroups", "X25519Kyber768Draft00");` on 2.0.65.Final
for my application, but when I trace the code to setCurvesList0, I get false instead of true. I would expect this named group to be configured when requested, which is consistent with what I get when running e.g. `./bssl client -curves X25519Kyber768Draft00 -connect localhost:4433 -debug`. Is there some additional property I need to set to get X25519Kyber768Draft00 recognized as a supported group?

**Comment by normanmaurer** [comment]
I don't think we respect this property in our implement... Maybe something we should do tho

**Comment by jmcrawford45** [comment]
@normanmaurer I see you added a related commit for this in #11660. I'm curious what else would need to be added as it seems the referenced PR adds this support. I also see the property parsed and passed all the way down to the native entry point, but setCurvesList0 fails on "X25519Kyber768Draft00", but passes on all other supported bssl NamedGroups.

**Comment by normanmaurer** [comment]
Oh so I already implemented it 🤣... maybe support is not present in the chromium-stable branch of boringssl ?

**Comment by jmcrawford45** [comment]
I'm able to spin up a server and client with chromium-stable@{'3 months ago'} and use them with e.g. `./bssl client -curves X25519Kyber768Draft00 -connect localhost:4433 -debug`

**Comment by normanmaurer** [comment]
I can have a look but it might take me a while ...

**Comment by jmcrawford45** [comment]
Thank you so much, please let me know if I can provide any additional context. I've been debugging this for a while, and my best guess at this point is that the ssl context created by Openssl.java excludes Kyber from the supported curves list field which causes the setCurve call to fail.

**Comment by jmcrawford45** [comment]
@normanmaurer I'm checking back in on this issue, please let me know if there's any way I can help with debugging.

**Comment by normanmaurer** [comment]
@jmcrawford45 unfortunately I had not time yet to investigate :/

**Comment by jmcrawford45** [comment]
@normanmaurer I'm checking back in on this issue, please let me know if there's any way I can help with debugging. Given the recent NIST [standardization](https://www.nist.gov/news-events/news/2024/08/nist-releases-first-3-finalized-post-quantum-encryption-standards), there is a growing interest in the ability to configure these algorithms.

**Comment by normanmaurer** [comment]
We now respect the property the right way
